feature-settings/src/main/java/cm/aptoide/pt/settings/presentation/SendFeedbackView.kt

```diff
@@ -252,12 +158,65 @@ fun SettingsViewContent(
         gradient = orangeGradient,
```

Comment:
Review the gradient in this button. It is different from the one in design.

Reply:
I've tried to use the value in the design but it is never the same, so i had left it as it is to be reviewed in another time